docs/concepts/actors.mdx

```diff
@@ -24,47 +19,41 @@ The academic definition of actors calls for them to be able to perform the follo
 
-## Stateless Actors
-wasmCloud actors are **stateless**, allowing hundreds or thousands of copies
-of an actor to be horizontally scaled across a cluster to meet compute demands.
-When an actor needs state, it obtains it easily through the use of a capability provider.
-By using capability providers, wasmCloud can ensure that no matter
-which instance of an actor is invoked, or where it's running in the cluster,
-it can still dynamically respond to the state of the larger application.
+### Stateless By Design
+
+wasmCloud actors are **stateless**, allowing hundreds or thousands of copies of an actor to be horizontally scaled across a cluster to meet compute demands. When an actor needs state, it obtains it easily through the use of a capability provider. By using capability providers, wasmCloud can ensure that no matter which instance of an actor is invoked, or where it's running in the cluster, it can still dynamically respond to the state of the larger application.
 
 Providers like the [concordance](https://github.com/cosmonic/concordance) event sourcing provider create a developer experience that feels very much like internally stateful actors, while still optimized for distributed systems.
 
-## Centralized Supervision
+### Centralized Supervision
+
 Actor supervision is a hallmark of the actor model. If you're used to systems like [Akka](https://akka.io/), where an application is one big supervision tree, and the only way to start an actor is from inside another actor, then wasmCloud's supervision model may seem awkward to you.
 
-wasmCloud actors do not supervise other actors.
-In a zero trust environment, allowing actors to spawn more is a security risk.
-wasmCloud hosts maintain the horizontal scale of actors with an entirely flat hierarchy.
+wasmCloud actors do not supervise other actors. In a zero trust environment, allowing actors to spawn others is a security risk. wasmCloud hosts maintain the horizontal scale of actors with an entirely flat hierarchy.
 
-## Actor Components
+### Actor Components
 
-An **_actor_ component** is the smallest unit of deployable, portable compute within the wasmCloud ecosystem. Actor components are small WebAssembly components that can handle messages delivered to them by the host runtime and can invoke functions on [capability providers](./capabilities), provided they have been granted the appropriate privileges.
+An **_actor_ component** is the smallest unit of deployable, portable compute within the wasmCloud ecosystem. Actor components are small WebAssembly components that can handle messages delivered to them by the host runtime and can invoke functions on [capability providers](./capabilities.mdx), provided they have been granted the appropriate privileges.
```

Comment:
First sentence probably deserves to stand on it's own.

Also it might be worth linking the phrase "WebAssembly components", since that means a very specific thing despite the generality of "component" as word